package.json

```diff
@@ -25,10 +25,10 @@
     "build:storybook": "yarn build && node .storybook/buildFontPreloads && build-storybook -s .storybook/static -c .storybook -o storybook_dist",
     "build:test": "cp envConfig/test.env .env && NODE_ENV=production webpack",
     "build:test:debug": "rm -rf build && awk '{sub(/LOG_DIR=.+/,\"LOG_DIR='log'\")}1' envConfig/test.env > .env && NODE_ENV=production webpack",
-    "cypress": "cypress run",
-    "cypress:interactive": "cypress open",
-    "cypress:3rdParty": "yarn cypress -- --project ./3rdPartyCypress/.",
-    "cypress:adhoc": "yarn cypress:interactive -- --project ./AdHocCypress/.",
+    "cypress": "cypress run --config-file cypress-simorgh.json",
```

Comment:
Thinking about it, we would have to change the infra e2e test scripts to use the correct config file. I had to do this because the new setup broke when a `cypress.json` exists within the root directory.